.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   extends: ['standard'],
   plugins: ['mocha'],
   env: {
+    'browser': true,
```

Comment:
this is needed for the events, window etc ?

Reply:
yes